velox/common/memory/MemoryPool.h

```diff
@@ -625,6 +625,10 @@ class MemoryPoolImpl : public MemoryPool {
     std::string callStack;
   };
 
+  void setDebugRegexFilter(const std::string& regexFilter) {
```

Comment:
Can we just pass debug filter on ctor for both memory manager and memory pool? Thanks!

Reply:
Discussed offline